fluentd/vendored_gem_src/fluent-plugin-prometheus/lib/fluent/plugin/in_prometheus_tail_monitor.rb

```diff
@@ -38,24 +38,31 @@ def configure(conf)
         @base_labels[key] = expander.expand(value)
       end
 
-      if defined?(Fluent::Plugin) && defined?(Fluent::Plugin::MonitorAgentInput)
-        # from v0.14.6
-        @monitor_agent = Fluent::Plugin::MonitorAgentInput.new
-      else
-        @monitor_agent = Fluent::MonitorAgentInput.new
-      end
+      @monitor_agent = Fluent::Plugin::MonitorAgentInput.new
     end
 
     def start
       super
 
       @metrics = {
```

Comment:
Missing: the high level value which is numebr of Log lines dropped is missing ... can you advice on the formula to calculate this from bellow metrics ?

Reply:
I saw this in @logloss but this is not exposed as metric (why?)

Reply:
It's not exposed as this is a raw value we should be able to calculate from exposing what is possible to collect and what is actually collected.

Reply:
restored back all changes done in vendored_gem_src as that is not supposed to be changed.